CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Android Card Form Release Notes
 
+## unreleased
+
+* Deprecate `useDialogForExpirationDateEntry()`
+* Changed default expiration date keyboard to use a system numeric keyboard
```

Comment:
Let's add a `Breaking changes` section to this part of the CHANGELOG since these changes to defaults are major updates.

Reply:
Yeah def